openmc/tallies.py

```diff
@@ -1963,8 +1964,12 @@ def _align_tally_data(self, other, filter_product, nuclide_product,
 
         Parameters
         ----------
+        other_old : openmc.Tally
+            The tally to outer product with this tally with the old tally
```

Comment:
perhaps re-phrase "with the old tally aligment" as ". This old tally retains the original alignment." or something? It is just unclear if the "with" refers to the "outer product" or to the "tally" in the sentence as currently structured. I just think that this is going to be confusing for future developers to understand and maintain.